crates/figma_import/src/toolkit_schema.rs

```diff
@@ -28,14 +28,18 @@ pub use crate::figma_schema::StrokeCap;
 /// Shape of a view, either a rect or a path of some kind.
 #[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
 pub enum ViewShape {
-    Rect,
+    Rect {
+        is_mask: bool,
```

Comment:
pity we have to replicate this common field; but there also isn't a convenient parent container to place it in. (no action needed)